workspace/all/minarch/minarch.c

```diff
@@ -6246,6 +6292,30 @@ int save_screenshot_thread(void* data) {
     return 0;
 }
 SDL_Thread* screenshotsavethread;
+static void Menu_screenshot(void) {
+	LOG_info("Menu_screenshot\n");
+
+	char rom_name[256];
+	getDisplayName(game.name, rom_name);
+	getAlias(game.path, rom_name);
+
+	time_t now = time(NULL);
+	struct tm *t = localtime(&now);
+	char buffer[100];
+	strftime(buffer, sizeof(buffer), "%Y-%m-%d-%H-%M-%S", t);
+
+	char png_path[256];
+	sprintf(png_path, SDCARD_PATH "/Screenshots/%s.%s.png", rom_name, buffer);
```

Comment:
Funny thing here, screenshots are PNGs but we always seem to write them as BMP files. This PR uses the correct filename for the user-facing screenshots.